pkg/test/framework/components/registryredirector/kube.go

```diff
@@ -119,9 +151,14 @@ func (c *kubeComponent) SetupTagMap(tagMap map[string]string) error {
 	if err != nil {
 		return err
 	}
-	_, err = client.Post(fmt.Sprintf("%s/admin/v1/tagmap", c.address), "application/json", bytes.NewBuffer(body))
+
+	err = retry.UntilSuccess(func() error {
+		_, err := client.Post(fmt.Sprintf("http://%s/admin/v1/tagmap", c.forwarder.Address()), "application/json", bytes.NewBuffer(body))
+		return err
+	}, retry.Delay(1*time.Second), retry.Timeout(20*time.Second))
```

Comment:
nit: `retry.BackoffDelay(100*time.Millisecond)` is probably slightly preferable. No need to sleep a full 1s  if its a quick call that should be ready soon

Reply:
Done